docs/components/pagination.md

```diff
@@ -0,0 +1,133 @@
+# Pagination
+
+**KPagination** is a component used for pagination.
+
+<template>
+  <KPagination :totalCount="300"/>
+</template>
+
+## Props
+### Total count - required
+A total number of items inside the paginated data source.
+
+```vue
+<KPagination :totalCount="50"/>
+```
+
+### Page sizes
+A list of page sizes that the user can choose from.
+
+Prop is an Array of numbers. The default page sizes are: [15, 25, 50, 75, 100].
+
+You can provide custom page sizes. The first one in the array will be the initial page size.
+
+<KPagination :totalCount="100" :pageSizes="[10, 20, 30, 40]"/>
+
+```vue
+<KPagination :totalCount="100" :pageSizes="[10, 20, 30, 40]"/>
+```
+
+### Neighbours
```

Comment:
```suggestion
### Neighbors
```

Sorry, can we change this to American spelling since most of our users are American 😅